libraries/SrcWrapper/inc/uart.h

```diff
@@ -168,6 +171,9 @@ struct serial_s {
 #define USART4_IRQn USART4_LPUART3_IRQn
 #define USART4_IRQHandler USART4_LPUART3_IRQHandler
 #endif /* LPUART3_BASE */
+#elif defined(STM32C0xx)
+#define USART4_IRQn USART3_4_IRQn
+#define USART4_IRQHandler USART3_4_IRQHandler
```

Comment:
This could by simplified by adding STM32C0xx with STM32G0xx
`#elif defined(STM32G0xx) || defined(STM32C0xx)`

Reply:
Done